### ISSUE TITLE
Unmounting component after beginDrag

### ISSUE THREAD

**Issue by beniutek** [react-dnd/react-dnd#634]
So yeah, the title says it all.
To be precise, I'm having issue, cause I don't know how to start, there is so far no issue with react-dnd about that.
I'm using Redux + React + react-dnd
Could anyone give me some advice about how to go along with unmounting component after we begin dragging them and preserve their state during that?

I have a list of components that can be a source and a target. When dragging a component I would like it to be unmounted from DOM, and then mounted back when I drop it. So far I have done everything regarding dragging dropping them without unmounting.

Basically my idea was to send an action `unmountItemFromList(itemId)` in the `beginDrag` and well make reducer do something with the state, but obviously, if I delete information about the item from the state then I will loose it alltogether so it's not a path to go.

Can I somehow affect whether the element is mounted or not during drag?

Spent some time googling that but couldn't find anything thats related to the react-dnd
